tests/Fixer/LanguageConstruct/IsNullFixerTest.php

```diff
@@ -52,6 +60,9 @@ public function testCorrectConfiguration()
     }
 
     /**
+     * @group legacy
+     * @expectedDeprecation Using "use_yoda_style" is deprecated and will be removed in 3.0. Use "yoda_style" fixer instead.
```

Comment:
this is the only test that will be left on 3.0. instead of adding legacy group, remove `$this->fixer->configure(['use_yoda_style' => true]);` line from it.

Reply:
I was planning to do it in the PR for 3.0, but I'm getting your point.

Reply:
And not this one, we will keep `testYodaFix`.

Reply:
you are right, my bad :+1: